app/api/dao/user.py

```diff
@@ -400,6 +401,18 @@ def get_achievements(user_id: int):
         achievements = [task for task in tasks if task.get("is_done")]
         return achievements
 
+    @staticmethod
+    def reset_password_token(token: str, expiration=config.BaseConfig.RESET_LINK_THRESHOLD):
+        from itsdangerous import URLSafeTimedSerializer, BadSignature
+        from run import application
+        password_reset_serializer = URLSafeTimedSerializer(application.config['SECRET_KEY'])
+        try:
+            email = password_reset_serializer.loads(
+            token, salt=application.config["SECURITY_RESET_SALT	"], max_age=expiration)
```

Comment:
```suggestion
            token, salt=application.config["SECURITY_RESET_SALT"], max_age=expiration)
```
you had a tab character there, I don't think that would have worked

Reply:
done 👍